packages/app-desktop/gui/MainScreen/commands/toggleMenuBar.ts

```diff
@@ -0,0 +1,20 @@
+import { CommandDeclaration, CommandRuntime } from '@joplin/lib/services/CommandService';
+import shim from '@joplin/lib/shim';
+import Setting from '@joplin/lib/models/Setting';
+import { _ } from '@joplin/lib/locale';
+
+export const declaration: CommandDeclaration = {
+	name: 'toggleMenuBar',
+	label: () => _('Toggle menu bar'),
+};
+
+export const runtime = (): CommandRuntime => {
+	return {
+		execute: async () => {
+			// Defensive code: macOS disallows hiding the menu bar, so ignore this Command
+			if (shim.isMac()) return;
```

Comment:
At this stage it's not necessary to handle this, since you already handle it where it matters

Reply:
Oh yup that makes sense. Removed in the newest commit.